src/components/DatasetOverviewTab/data/definitions.json

```diff
@@ -0,0 +1,50 @@
+[
+    {
+      "pdc": "Modified",
+      "definition": "Most recent date on which the dataset was changed, updated or modified"
+    },
+    {
+      "pdc": "Released",
+      "definition": "xxx"
+    },
+    {
+      "pdc": "Issued",
+      "definition": "Dare of formal issuance\" Project Open Data uses lable \"Release"
+    },
+    {
+      "pdc": "Publisher",
+      "definition": "The publishing entity and optionally their parent organization(s)"
+    },
+    {
+      "pdc": "Identifier",
+      "definition": "A unique identifier for the dataset or API as maintained within an Agency catalog or database"
+    },
+    {
+      "pdc": "Contact",
+      "definition": "Contact name for the asset"
+    },
+    {
+      "pdc": "Contact Email",
+      "definition": "Contact name for the asset"
+    },
+    {
+      "pdc": "Public Access Level",
+      "definition": "The degree to which this dataset could be made pubicly-available. \nChoices:\npublic (data asset is or could be made publicly available to all without restrictions), \nrestricted public (Data aset is available under certain use restrictions) or \nnon-public (Data asset is not available to members of the public"
+    },
+    {
+      "pdc": "Homepage URL",
```

Comment:
We should probably dynamically hide values that are not shown on any particular site. For example, this field does not display on data.medicaid, and so this option should also not display in the help drawer for that site.

Reply:
We should probably also order these entries dynamically to match the order of the downstream site so it is not confusing.

Reply:
addressed:

https://github.com/GetDKAN/cmsds-open-data-components/pull/239/commits/1118d099fbdb3a814209d8f3e59b06d4681d0dbd

![Screenshot 2024-08-16 at 3 20 05 PM](https://github.com/user-attachments/assets/26d1c0f1-67ca-46dd-82c0-f54f16ec0a48)